### DISCUSSION TITLE
Welcome to OpenRVDAS Discussions!

### DISCUSSION THREAD

**Discussion by davidpablocohn** [OceanDataTools/openrvdas#366] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pauldavidmena** [comment]
My name is Paul Mena. I am an application administrator at Woods Hole Oceanographic Institution in Massachusetts. Most of what I do entails moving data around, as well as supporting the infrastructure responsible for this movement. WHOI is in the process of migrating its research fleet from a legacy data acquisition application to OpenRVDAS. So far, we've noted that it installs very easily and starts up without issues. Now it's a matter of integration.

**Reply by davidpablocohn** [reply]
Hi Paul - great to hear! Please give a holler here, or by email, or via our woefully-underused Discord ( https://discord.gg/mccbXFXk ) with any questions.
